<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDetailInfo</class>
 <widget class="QWidget" name="WidgetDetailInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget {border-image: url(:/UI/back/targetback.png); }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="44,152,85,265,20">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>思源黑体 CN</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>详细信息</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="styleSheet">
            <string notr="true">border-image: url(:/UI/back/blankh.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="MultiLargeScreen.qrc">
             <normaloff>:/UI/button/close2x.png</normaloff>:/UI/button/close2x.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetVideo" native="true">
        <property name="styleSheet">
         <string notr="true">#widget_4{border-image: url(:/UI/back/image2.png);}</string>
        </property>
        <widget class="LabelClick" name="lblFull">
         <property name="geometry">
          <rect>
           <x>268</x>
           <y>132</y>
           <width>14</width>
           <height>14</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/UI/icon/videozoomin.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/UI/back/blankv.png);color: rgb(255, 255, 255);;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>2</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>2</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/UI/image/spliteType2.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,5,2,5,1" columnstretch="1,10,10,1">
           <item row="1" column="2">
            <widget class="QPushButton" name="btnPreset2">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton::menu-indicator {
    subcontrol-position: right center;
    subcontrol-origin: padding;
    left: -2px;}</string>
             </property>
             <property name="text">
              <string>预置位2</string>
             </property>
             <property name="icon">
              <iconset theme="circle_tr2x.png" resource="MultiLargeScreen.qrc">
               <normaloff>:/UI/icon/circle_tr2x.png</normaloff>
               <selectedon>:/UI/icon/circle2x.png</selectedon>:/UI/icon/circle_tr2x.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btnPreset3">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton::menu-indicator {
    subcontrol-position: right center;
    subcontrol-origin: padding;
    left: -2px;}</string>
             </property>
             <property name="text">
              <string>预置位3</string>
             </property>
             <property name="icon">
              <iconset resource="MultiLargeScreen.qrc">
               <normaloff>:/UI/icon/circle_tr2x.png</normaloff>
               <selectedon>:/UI/icon/circle2x.png</selectedon>:/UI/icon/circle_tr2x.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnPreset1">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton::menu-indicator {
    subcontrol-position: right center;
    subcontrol-origin: padding;
    left: -2px;}</string>
             </property>
             <property name="text">
              <string>预置位1</string>
             </property>
             <property name="icon">
              <iconset resource="MultiLargeScreen.qrc">
               <normaloff>:/UI/icon/circle_tr2x.png</normaloff>
               <selectedon>:/UI/icon/circle2x.png</selectedon>:/UI/icon/circle_tr2x.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="btnPreset4">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton::menu-indicator {
    subcontrol-position: right center;
    subcontrol-origin: padding;
    left: -2px;}</string>
             </property>
             <property name="text">
              <string>预置位4</string>
             </property>
             <property name="icon">
              <iconset resource="MultiLargeScreen.qrc">
               <normaloff>:/UI/icon/circle_tr2x.png</normaloff>
               <selectedoff>:/UI/icon/circle_tr2x.png</selectedoff>
               <selectedon>:/UI/icon/circle2x.png</selectedon>:/UI/icon/circle_tr2x.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelNext">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/UI/icon/more2x.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout" columnstretch="5,1,12">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>联系电话:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>设备名称:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>思源黑体 CN Normal</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>负责人:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="lblPhone">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>13800138003</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>思源黑体 CN Normal</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>设备IP:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>设备类型:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="lblContacts">
             <property name="font">
              <font>
               <family>思源黑体 CN Normal</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>李四</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lblName">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>北辰路大屯路口东南-北1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>设备状态:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="lblIp">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>172.10.10.234</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="lblStatus">
             <property name="font">
              <font>
               <family>思源黑体 CN Normal</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>正常</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="lblType">
             <property name="font">
              <font>
               <family>思源黑体 CN</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>高点</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LabelClick</class>
   <extends>QLabel</extends>
   <header location="global">labelclick.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MultiLargeScreen.qrc"/>
 </resources>
 <connections/>
</ui>
